### ISSUE TITLE
Allow side padding color to be changed for webtoons

### ISSUE THREAD

**Issue by Adoridenn** [Jays2Kings/tachiyomiJ2K#1453]
### Describe your suggested feature

Have the ability to change the side padding color to black, white, and grey, for webtoons. Match manga color would also be nice to have.

### Other details

I've experienced burn in with heavy use (5-10+ hours daily) of reading webtoons with 25% padding, over a period of 2 weeks on a new tab s8 ultra. Right now the burn in is extremely subtle, 2 thin, barely visible lines are in the middle of the screen when viewing full screen solid colors. The black padding creates a border of white/black in webtoons, where burn in can happen right at the border. May be a faulty display, but this feature can definitely help mitigate this issue in the future.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an request for a feature in the original Tachiyomi, I am opening because multiple versions of TachiyomiJ2K have been released without this feature.
- [X] If this is an issue with an extension, or a request for an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[1.5.9](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
Does setting the display settings background to smart not work?

**Comment by Adoridenn** [comment]
> Does setting the display settings background to smart not work?

No, side padding and background color seem to be two separate settings, setting background color doesn't effect side padding. When reading a webtoon, with reader mode set to webtoon or continuous, with "side padding" set to 5% or more,  changing background color doesn't change the side padding color at all, it is pure black. If reading mode is set to left to right, or anything besides continuous and webtoon, then the background color changes according to background color settings. 

Need a new setting that allows changing the side padding color itself. As seen in screenshots, the color only works on background and not side padding. Black,white,grey and smart would be nice, but even just white color would help since most webtoons have a white background already, so there wouldn't be as harsh a border where webtoon and padding meet.

Side padding 25% with webtoon mode
![Screenshot_20221115_081822_TachiyomiJ2K](https://user-images.githubusercontent.com/4638704/201930775-62fcff58-c5a6-4f3d-be95-f6dfce11cf67.jpg)

Normal mode
![Screenshot_20221115_081925_TachiyomiJ2K](https://user-images.githubusercontent.com/4638704/201930977-f81f0dca-5771-47b5-8240-551c481016c9.jpg)
